kodilla-stream/src/main/java/com/kodilla/stream/StreamMain.java

```diff
@@ -1,30 +1,24 @@
 package com.kodilla.stream;
 
-import com.kodilla.stream.beautifier.PoemBeautifier;
-import com.kodilla.stream.lambda.ExpressionExecutor;
-import com.kodilla.stream.reference.FunctionalCalculator;
+import com.kodilla.stream.forumuser.Forum;
+import com.kodilla.stream.forumuser.ForumUser;
+
+import java.time.LocalDate;
+import java.util.Map;
+import java.util.stream.Collectors;
 
 public class StreamMain {
     public static void main(String[] args) {
-        ExpressionExecutor expressionExecutor = new ExpressionExecutor();
-
-        System.out.println("Calculating expressions with lambdas");
-        expressionExecutor.executeExpression(10, 5, (a, b) -> a + b);
-        expressionExecutor.executeExpression(10, 5, (a, b) -> a - b);
-        expressionExecutor.executeExpression(10, 5, (a, b) -> a * b);
-        expressionExecutor.executeExpression(10, 5, (a, b) -> a / b);
-
-        System.out.println("Calculating expressions with method references");
-        expressionExecutor.executeExpression(3, 4, FunctionalCalculator::multiplyAByB);
-        expressionExecutor.executeExpression(3, 4, FunctionalCalculator::addAToB);
-        expressionExecutor.executeExpression(3, 4, FunctionalCalculator::subBFromA);
-        expressionExecutor.executeExpression(3, 4, FunctionalCalculator::divideAByB);
+        Forum forum = new Forum();
+        Map<Integer, ForumUser> theResultMapOfUsers = forum.getList().stream()
+                .filter(forumUser -> forumUser.getUserSex() == 'M')
+                .filter(forumUser -> forumUser.getUserDateOfBirth().isBefore(LocalDate.of(1998, 10, 21)))
+                .filter(forumUser -> forumUser.getUserNumberOfPosts() > 0)
+                .collect(Collectors.toMap(ForumUser::getUserSignature, forumUser -> forumUser));
 
-        PoemBeautifier poemBeautifier = new PoemBeautifier();
-        System.out.println("Beautifying text commencing: ");
-        poemBeautifier.beautify("hello world", text -> "ABC" + text + "ABC");
-        poemBeautifier.beautify("grażynka", String::toUpperCase);
-        poemBeautifier.beautify("Mućka", text -> "Krasula");
-        poemBeautifier.beautify("WYŁĄCZ CAPS LOCKA!!!", String::toLowerCase);
+        System.out.println("Number of Elements: " + theResultMapOfUsers.size());
+        theResultMapOfUsers.entrySet().stream()
+                .map(entry -> entry.getKey() + ": " + entry.getValue())
+                .forEach(System.out::println);
```

Comment:
te zmiany naleza do innego zadania. Prawdopodobnie wyszedles z 7.3, tworzac 7.4, zamiast z Mastera

Reply:
To do zapamietania przy nastepnych zadaniach